components/Dashboard/CardGrid.js

```diff
@@ -17,6 +14,14 @@ export function CardGrid() {
     }
     fetchData();
   }, []);
+  useEffect(() => {
+    async function fetchData() {
+      const response = await fetch("/api/meetings");
+      const data = await response.json();
+      setMeetings(data);
+    }
+    fetchData();
+  }, []);
```

Comment:
Fetching can always fail! You need to wrap this into a `try / catch` block and handle errors somehow (e.g. open an alert).